src/Components/Material/Materials.jsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { Icon } from "@iconify/react";
+import style from "./Materials.module.scss";
+import folderpic from "../../Assets/Images/folder.png";
```

Comment:
Once the image is moved to public folder. Then use it as 

```html

<img src="/images/material/folder.png" alt="Folder"/>

```

or for this component, change the image to "/images/material/folder.png" instead of folderpic